graphs/gen/batagelj_brandes.go

```diff
@@ -0,0 +1,355 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// The functions in this file are random graph generators from the paper
+// by Batagelj and Brandes http://algo.uni-konstanz.de/publications/bb-eglrn-05.pdf
+
+package gen
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+
+	"github.com/gonum/graph"
+	"github.com/gonum/graph/simple"
+)
+
+// Gnp constructs a graph in the destination, dst, of order n. Edges between nodes
+// are formed with the probability, p. If src is not nil it is used as the random source
```

Comment:
Ok. Note that the same comma should also be added before "otherwise" in the other comments as well.